### ISSUE TITLE
Run github actions depending on path

### ISSUE THREAD

**Issue by surister** [crate/cratedb-sqlparse#39]
Right now any PR triggers a full run of tests, for both python and javascript, ideally we should only ran python tests if the python target changes for example.

https://stackoverflow.com/questions/63822219/how-to-run-github-actions-workflow-only-if-the-pushed-files-are-in-a-specific-fo

**Comment by amotl** [comment]
Good idea. Please see [cratedb-examples' workflows](https://github.com/crate/cratedb-examples/tree/main/.github/workflows) for more inspirations and blueprints.
